src/polkitbackend/polkitd.c

```diff
@@ -38,9 +38,12 @@ static GMainLoop              *loop = NULL;
 static gint                    exit_status = EXIT_FAILURE;
 static gboolean                opt_replace = FALSE;
 static gboolean                opt_no_debug = FALSE;
+static gint                    opt_log_level = (gint) LOG_LEVEL_ERROR;
 static GOptionEntry            opt_entries[] = {
   {"replace", 'r', 0, G_OPTION_ARG_NONE, &opt_replace, "Replace existing daemon", NULL},
-  {"no-debug", 'n', 0, G_OPTION_ARG_NONE, &opt_no_debug, "Don't print debug information", NULL},
+  {"no-debug", 'n', 0, G_OPTION_ARG_NONE, &opt_no_debug, "Don't print debug information to stderr and stdout", NULL},
+  {"log-level", 'l', 0, G_OPTION_ARG_INT, &opt_log_level, "Set a level of logging. Values (0-3): errors, warnings, notifications, verbose.",
+          "[0,1,2,3]"},
```

Comment:
Oh yeah, especially *numerical* values different than syslog would be really confusing. Please don't do this. And names too. Please just reuse the standard values.